### PR TITLE
Update solution-4.tex

### PR DESCRIPTION
**Pull request by danteorbit**
6c. not (a => b) to (a and not b)
6f. not (a or b) to (not a and not b)

### CODE REVIEW

**Comment by kyrylo** [comment]
Thanks for the PR! I really can't remember what this was about, but I trust you 😄

**Comment by kyrylo** [comment]
Hmm, okay so this if from the assignment. Did they change it? Check the PDF in the same directory.

**Comment by danteorbit** [comment]
image

**Comment by danteorbit** [comment]
Current solutions from course tutorial 🫡

File: lecture-4-equivalence/solution-4.tex [hunk 1]
```diff
@@ -83,10 +83,10 @@ \section{}
 \begin{enumerate}[(a)]
 \item 34,159 is not a prime number.
 \item Roses are not red and violets are not blue.
```
Comment (kyrylo) [inline lecture-4-equivalence/solution-4.tex:85]: @dantevangelista

Should this be `Roses are not red or violets are not blue.`? (changed `and` to `or`)

Comment (danteorbit) [reply lecture-4-equivalence/solution-4.tex:85]: yes

**Pull request merged**